artifactory/utils/filetree.go

```diff
@@ -86,15 +91,32 @@ func (dn *dirNode) strings() []string {
 		subDirIndex++
 	}
 	fileIndex := 0
-	for fileName := range dn.fileNames {
+
+	// Sort File names inside each sub dir
+	var fileNamesSorted []string
```

Comment:
Any reason for introducing a new variable and not sorting the original dn.fileNames?